### PR TITLE
add readme to undraw folder

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

Close https://github.com/nextcloud/social/issues/1394#issuecomment-1127620297

### CODE REVIEW

**Comment by CarlSchwan** [comment]
I would prefer a .reuse/dep5 file since this is a standard to declare licensing information. Example https://github.com/nextcloud/profiler/blob/master/.reuse/dep5

Something like this should be enough

```
Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
Upstream-Name: Nextcloud Social
Upstream-Contact: Nextcloud <https://nextcloud.com/contact/>
Source: https://github.com/nextcloud/social

Files: img/undraw/*
Copyright: Katerina Limpitsouni
License: AGPL-3.0-or-later
```

**Comment by szaimen** [comment]
> I would prefer a .reuse/dep5 file since this is a standard to declare licensing information. Example https://github.com/nextcloud/profiler/blob/master/.reuse/dep5
> 
> Something like this should be enough
> 
> ```
> Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
> Upstream-Name: Nextcloud Social
> Upstream-Contact: Nextcloud <https://nextcloud.com/contact/>
> Source: https://github.com/nextcloud/social
> 
> Files: img/undraw/*
> Copyright: Katerina Limpitsouni
> License: AGPL-3.0-or-later
> ```

Fine by me! Let me edit the PR... 
(Never heard of this so adding a md was the best I was able to come up with) :D

**Comment by szaimen** [comment]
> > Something like this should be enough
> > ```
> > Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
> > Upstream-Name: Nextcloud Social
> > Upstream-Contact: Nextcloud <https://nextcloud.com/contact/>
> > Source: https://github.com/nextcloud/social
> > 
> > Files: img/undraw/*
> > Copyright: Katerina Limpitsouni
> > License: AGPL-3.0-or-later
> > ```
> 
> Fine by me! Let me edit the PR...

done

File: .reuse/dep5 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Nextcloud Social
+Upstream-Contact: Nextcloud <https://nextcloud.com/contact/>
+Source: https://github.com/nextcloud/social
+
+Files: img/undraw/*
+Copyright: Katerina Limpitsouni
+License: AGPL-3.0-or-later
```
Comment (CarlSchwan) [inline .reuse/dep5:8]:
```suggestion
License: AGPL-3.0-or-later
Comment:  The undraw illustrations are released with a non-free license and thus not AGPL-compatible but we have the illustrators permissions to use these 8 images in this project. See https://github.com/nextcloud/social/issues/277#issuecomment-508480655 and below.
```

Comment (szaimen) [reply .reuse/dep5:8]: done

**Comment by szaimen** [comment]
Seems like the tests are not running=

**Comment by szaimen** [comment]
Can we merge this?

**Pull request merged**